includes/lite-site/class-lite-site.php

```diff
@@ -442,6 +442,9 @@ public static function clean_content( $content ) {
 		// First remove figures and their contents (including images and captions).
 		$content = preg_replace( '/<figure.*?>.*?<\/figure>/s', '', $content );
 
+		// Remove script tags.
+		$content = preg_replace( '/<script.*?>.*?<\/script>/s', '', $content );
```

Comment:
It works, but is case sensitive, so this will still be printed:
```js
<SCRIPT>alert(1)</SCRIPT>
```

We should add the `i` modifier to the regex to prevent this:
```suggestion
		$content = preg_replace( '/<script.*?>.*?<\/script>/is', '', $content );
```

Reply:
Thanks. Fixed that in the `figure` tag a few lines above as well in https://github.com/Automattic/newspack-plugin/pull/4093/commits/b30b90bfc57e1f9ddea86b85a3f10c1e07ea032e